Users/Glob_Patterns.rst

```diff
@@ -1,6 +1,47 @@
 Glob - Extended unix style pathname expansion
 =============================================
 
+Using Glob Patterns
+-------------------
+
+Suppose you want `SpaceConsistencyBear` to perform an analysis on a file
+``first.c``, you should use the command:
+::
+
+    coala --files=src/first.c --bears=SpaceConsistencyBear --save
+
+.. note::
+
+    If you don't know the fuctions of a bear or how to perform the analysis
+    with a bear, you should go through `Tutorial
+    <http://coala.readthedocs.io/en/latest/Users/Tutorial.html>`_ first.
+
+Now, if you want all the ``.c`` files in a specific directory to be analysed,
+you can take help of glob patterns.
+::
+
+    coala --files='src/*.c' --bears=SpaceConsistencyBear --save
+
+Here, ``*.c`` matches all ``.c`` files in the specified directory.
+Going further, if you want all ``.c`` as well as ``.java`` files to
+be analysed:
+::
+
+    coala --files='src/*.(java|c)' --bears=SpaceConsistencyBear --save
+
+If you want your ``files`` argument to match all directories and
+subdirectories, you can use ``**`` glob pattern for that.
+::
+
+    files = **/*.c
+
+.. note::
+
+    While using ``files = **/*.c`` , since we have used ``\`` in the glob
+    pattern so all ``.c`` files at least one subdirectory below the root
```

Comment:
I don't think using a `so` is valid when you begin a phrase with `since`? Not sure, others can chip in.